engine/schema/src/main/java/com/cloud/hypervisor/dao/HypervisorCapabilitiesDaoImpl.java

```diff
@@ -106,4 +107,16 @@ public Boolean isVmSnapshotEnabled(HypervisorType hypervisorType, String hypervi
         HypervisorCapabilitiesVO result = getCapabilities(hypervisorType, hypervisorVersion);
         return result.getVmSnapshotEnabled();
     }
+
+    @Override
+    public List<HypervisorType> getHypervisorsWithDefaultEntries() {
```

Comment:
Is there a possibility of high frequency calls to this method? If so we can perhaps optimize query calls with some caching? If not LGTM

Reply:
Thanks @anuragaw, have refactored it to invoke this only once